src/pages/Domains/Domains.tsx

```diff
@@ -0,0 +1,57 @@
+//- React Imports
+import { FC } from 'react';
+
+//- Types Imports
+import { Metadata } from '../../lib/types/metadata';
+
+//- Features Imports
+import SubdomainTable from '../../features/view-subdomains/SubdomainTable/SubdomainTable';
+import SubdomainMetrics from '../../features/view-subdomains/SubdomainMetrics/SubdomainMetrics';
+import DomainPreview from '../../features/domain-preview/DomainPreview';
+
+//- Library Imports
+import { Domain, DomainMetrics, TokenPriceInfo } from '@zero-tech/zns-sdk';
+
+type DomainsContainerProps = {
+	route: string;
+	domain: Domain;
+	metrics: DomainMetrics;
+	subdomainData: Domain[];
+	domainMetadata: Metadata;
+	paymentTokenInfo: TokenPriceInfo;
+	isSubdomainDataLoading?: boolean;
+};
+
+const DomainsContainer: FC<DomainsContainerProps> = ({
+	route,
+	domain,
+	metrics,
+	subdomainData,
+	domainMetadata,
+	paymentTokenInfo,
+	isSubdomainDataLoading,
+}) => {
+	const isRoot = route === 'wilder';
```

Comment:
Could make this more generic by doing `route is longer than 0 characters, and route does not contain a .`

Reply:
nice, updated 👍

Reply:
Assuming the 'root' path will always be 'wilder', I've added the following
```
	const isRoot = route.split('.').length === 1 && !route.includes('.');
```

Not in scope right now, but will we need to handle network switching in the near future?